### DISCUSSION TITLE
stoicism/memento_mori/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#10] [General]
# stoicism/memento_mori/

https://shanenull.com/stoicism/memento_mori/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/maranasati/

**Reply by shane0** [reply]
https://www.ssa.gov/oact/STATS/table4c6.html

**Reply by shane0** [reply]
Age	Target HR Zone 50-85%	

Age-predicted Maximum Heart Rate

20 years	100-170 bpm	200 bpm
30 years	95-162 bpm	190 bpm
35 years	93-157 bpm	185 bpm
40 years	90-153 bpm	180 bpm
45 years	88-149 bpm	175 bpm
50 years	85-145 bpm	170 bpm
55 years	83-140 bpm	165 bpm
60 years	80-136 bpm	160 bpm
65 years	78-132 bpm	155 bpm
70 years	75-128 bpm	150 bpm

**Comment by shane0** [comment]
# earthquake maps

- aka the map aliens use to lacate the dumb humans like me who live on fault lines
- https://glowy-earthquakes.glitch.me/
